src/uu/cksum/src/cksum.rs

```diff
@@ -442,8 +451,7 @@ pub fn uu_app() -> Command {
             Arg::new(options::UNTAGGED)
                 .long(options::UNTAGGED)
                 .help("create a reversed style checksum, without digest type")
-                .action(ArgAction::SetTrue)
-                .overrides_with(options::TAG),
+                .action(ArgAction::SetTrue),
```

Comment:
~~Why? They still override each other, and even if "both are set" then the binary flag still is correct:~~

```console
$ ../gnu/src/cksum --untagged --tag -b --algo=md5 target/debug/coreutils
MD5 (target/debug/coreutils) = f70b38fd8e152eda7da97dd6dd0c7d76
$ ../gnu/src/cksum --tag --untagged -b --algo=md5 target/debug/coreutils
f70b38fd8e152eda7da97dd6dd0c7d76 *target/debug/coreutils
$ ../gnu/src/cksum --tag --untagged --algo=md5 target/debug/coreutils
f70b38fd8e152eda7da97dd6dd0c7d76  target/debug/coreutils
```

EDIT: Ah, this is fixed by a later commit.